src/common/EndpointNumber.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2022 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TlvUInt16 } from "../tlv/TlvNumber.js";
+import { TlvWrapper } from "../tlv/TlvWrapper.js";
+import { MatterCoreSpecificationV1_0 } from "../spec/Specifications.js";
+
+/**
+ * A Endpoint Number is a 16-bit number that that indicates an instance of a device type.
```

Comment:
![Bildschirm­foto 2022-12-09 um 22 23 00](https://user-images.githubusercontent.com/11976694/206798228-f9ba70b9-c598-4810-b4cf-9d11d2e85e6a.png)

The naming was 1:1 done as in the specs because we discussed the naming topic and decided that it is best to stay as near as possible to the specs

Reply:
**Agreed with Apollon77**: it is easier to follow the spec if the naming is matching.
**Note**: I don't like that name either and many other names in the specs....